Update build an application with a certificate with MSAL-Node

salman90 opened the pull request:
We have a sample in MSAL-Node that authorizes the user with a certificate. Please look at the [sample ](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/samples/msal-node-samples/auth-code-with-certs)and the [docs ]( https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-node/docs/certificate-credentials.md)for more information.

salman90 committed:
Update build an application with a certificate with MSAL-Node

We have a sample in MSAL-Node that authorizes the user with a certificate. Please look at the sample https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/samples/msal-node-samples/auth-code-with-certs and the docs https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-node/docs/certificate-credentials.md for more information.

sameerag reviewed (commented):
Looks good. Couple of comments:
* Are we linking to msal-node docs that elaborate the usage?
* Can we do the same for `client assertions`?

salman90 committed:
updated scenario-daemon-app-configuration.md

Added authorization with client assertion to the tutorial. Also improved consistency by setting the configuration object values from the .env file.

EmLauber reviewed (approved):
Looks good to me

jmprieur reviewed (approved):
LGTM

#sign-off

ktoliver committed:
PR review edit, line spacing and punctuation
